src/sync.ts

```diff
@@ -0,0 +1,905 @@
+import * as fs from "fs-extra";
```

Comment:
Is there any code written ( in terms of logic improvement and new code)  in this file except improvements like async-await and tslint ?
If yes, it would be great if you mark those lines.